timely_beliefs/visualization/graphs.py

```diff
@@ -0,0 +1,283 @@
+from typing import Tuple, Union
+
+import altair as alt
+
+from timely_beliefs.visualization import selectors
+
+
+def base_chart(source, belief_horizon_unit: str) -> alt.Chart:
+    return (
+        alt.Chart(source)
+        .encode(
+            x=alt.X(
+                "event_start",
+                axis=alt.Axis(labelFlush=False),
+                scale={"domain": selectors.time_selection_brush.ref()},
+                title="Event start",
+            ),
+            color=selectors.source_color_or(selectors.idle_color),
+            tooltip=[
+                alt.Tooltip(
+                    "event_start:T",
+                    timeUnit="yearmonthdatehoursminutes",
+                    title="Event start",
+                ),
+                alt.Tooltip(
+                    "event_end:T",
+                    timeUnit="yearmonthdatehoursminutes",
+                    title="Event end",
+                ),
+                alt.Tooltip("expected_value:Q", title="Expected value", format=".2f"),
+                alt.Tooltip(
+                    "belief_time:T",
+                    timeUnit="yearmonthdatehoursminutes",
+                    title="Belief time",
+                ),
+                alt.Tooltip(
+                    "belief_horizon:Q",
+                    title="Belief horizon (%s)" % belief_horizon_unit,
+                ),
+                alt.Tooltip("source", title="Source"),
+            ],
+        )
+        .properties(width=550, height=200)
+    )
+
+
+def time_series_chart(
```

Comment:
This is also a base chart of kinds, but I find the name "time_series_base_chart" would be more fitting for the "base" chart you create with above method, as it makes sure it has a time-bases x axis and proper tooltips for times.

This one, seems to be responsible for adding actual graphs (or "values") for being displayed in the base time series chart.
So, as usual, I'd propose to rename these functions accordingly.